### DISCUSSION TITLE
v2.0.0-alpha.1

### DISCUSSION THREAD

**Discussion by KevinVandy** [KevinVandy/mantine-react-table#169] [Announcements]
## What's Changed
* Row Pinning feature by @KevinVandy 
* V2 by @DenisBessa in https://github.com/KevinVandy/mantine-react-table/pull/124
* Module css by @meesvandongen in https://github.com/KevinVandy/mantine-react-table/pull/126
* do some more icon -> leftSection by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/127
* convert a couple sx to style as a test of pr process by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/125
* fix: updated docs index pages, removed the sx props by @enyelsequeira in https://github.com/KevinVandy/mantine-react-table/pull/128
* UI fixes v2 by @enyelsequeira in https://github.com/KevinVandy/mantine-react-table/pull/129
* chore: updating index page removing sx props started tables by @enyelsequeira in https://github.com/KevinVandy/mantine-react-table/pull/130
* start work on fixing action icons... by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/131
* Refactored MRT_TabelBodyRow to CSS module by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/132
* Toggle row action menu by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/133
* Footer components by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/134
* Head components by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/135
* Input components by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/137
* refactor: column utils to CSS modules by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/138
* fix comment style on columns.util by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/139
* re-add getprimaryshade and getprimarycolor by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/140
* fix storybook runtime errors by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/141
* replace all instanceOf Function ternaries with a common function by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/142
* add base MRT_ActionIcon by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/143
* move complex styles out of style tag into module by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/145
* upgrade to 7.1 and fix the striped demo by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/146
* Fix row styling by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/147
* fix layout of pagination select by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/150
* updated all buttons to use css modules and className convention by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/151
* add modules to FilterOptionsMenu, ShowHideColumnsMenu and remove isSubMenu from ShowHideColumnsMenu by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/152
* Refactor Toolbars to use CSS modules by @alessandrojcm in https://github.com/KevinVandy/mantine-react-table/pull/153
* Solves build errors by @DenisBessa in https://github.com/KevinVandy/mantine-react-table/pull/154
* convert more inline styles to classes and modules by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/160
* Fix typos in sv.ts by @Crotalus in https://github.com/KevinVandy/mantine-react-table/pull/159
* get rid of a bunch more inline styles by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/163
* remove last uses of style={{ that i can by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/164
* fix toolbar background color on light mode by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/165
* get build working by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/166
* fix a couple bugs in my own v2 changes by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/167
* rethink how to style action icons using --ai vars by @naughton in https://github.com/KevinVandy/mantine-react-table/pull/168

## New Contributors
* @DenisBessa made their first contribution in https://github.com/KevinVandy/mantine-react-table/pull/124
* @meesvandongen made their first contribution in https://github.com/KevinVandy/mantine-react-table/pull/126
* @naughton made their first contribution in https://github.com/KevinVandy/mantine-react-table/pull/127
* @alessandrojcm made their first contribution in https://github.com/KevinVandy/mantine-react-table/pull/132
* @Crotalus made their first contribution in https://github.com/KevinVandy/mantine-react-table/pull/159

**Full Changelog**: https://github.com/KevinVandy/mantine-react-table/compare/v1.2.1...v2.0.0-alpha.1

<hr /><em>This discussion was created from the release <a href='https://github.com/KevinVandy/mantine-react-table/releases/tag/v2.0.0-alpha.1'>v2.0.0-alpha.1</a>.</em>

**Comment by KevinVandy** [comment]
We have started publishing alphas that are compatible with Mantine V7, though I think we are still a bit away from production ready. Try at your own risk with the understanding that it is not ready yet.

Join the [discord](https://discord.gg/5wqyRx6fnm) if you are interested in helping with the Mantine v7 upgrade or voicing your opinions on our active development. It's been great to be getting help from the Mantine community so far.

**Reply by finom** [reply]
What are the breaking changes besides moving to Mantine v7? I've checked the changelog and there seems to be no API updates, am I correct?
